bands/templates/emails/band_stage_plot_reminder.txt

```diff
@@ -2,6 +2,6 @@
 
 In order to prepare for your performance at this upcoming {{ c.EVENT_NAME }}, we need you to upload an exact description of your desired stage layout, which you can do at {{ c.URL_BASE }}/bands/stage_plot?band_id={{ band.id }}
 
-Failure to provide exact stage directions may result in you not receiving the setup you expect or want.  Please provide this description no later than {{ c.STAGE_AGREEMENT_DEADLINE|datetime_local }}.
+Failure to provide exact stage directions may result in you not receiving the setup you expect or want.  Please provide this description no later than {{ c.BAND_STAGE_PLOT_DEADLINE|datetime_local }}.
```

Comment:
I didn't genericize the email templates because I doubt one template will work for more than one group type. Instead I changed automated_emails to check and make sure the group type == c.BAND.